docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| napari term        | Description                                                                                                                                                                                                                                                                                                                                 |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
+| color map          | An array of colors used to map pixel data to the actual color values.                                                                                                                                                                                                                                                                       |
+| dock widget        | A widget that can be attached to the main napari viewer at a specified position.                                                                                                                                                                                                                                                            |
+| hook               | Hook is the place within napari where we “promise” to call functions created by external developers and installed by the user.                                                                                                                                                                                                              |
```

Comment:
* Why are there quotes around the word "promise"? I don't think those need to be there.
* Possibly useful links:
    * https://napari.org/docs/dev/plugins/hook_specifications.html
    * https://en.wikipedia.org/wiki/Hooking

Reply:
an example would be useful here e.g. we use a "hook" in the Viewer's open method to try plugin file readers before using the builtins